src/main/java/edu/umich/ctools/sectionsUtilityTool/SISPollingThread.java

```diff
@@ -264,10 +317,36 @@ public BodyPart getAttachment(SISDataHolderForEmail data) throws MessagingExcept
 		BodyPart attachment = new MimeBodyPart();
 		attachment.setDataHandler(new DataHandler(new ByteArrayDataSource(data.getSisEmailData().getBytes(),
 				Utils.CONSTANT_MIME_TEXT_CSV)));
-		if (data.getSisProcessType().equals(SISUploadType.ADD_SECTIONS)) {
-			attachment.setFileName(SISUploadType.ADD_SECTIONS.getDescription() + "_"+ data.getCourseId()+".csv");
-		}
+		attachment.setFileName(data.getSisProcessType().getDescription() + "_" + data.getCourseId() + ".csv");
+
 		return attachment;
 
 	}
+
+	// defaults to 5 attempts (if property goes missing/bad in ccm.properties) to retry polling before giving up
+	private int getPollingAttempt() {
+		return getIntegerValueOfProperty(Utils.SIS_POLLING_ATTEMPTS);
+	}
+
+	//defaulting to every minute polling if the property goes missing/bad in ccm.properties
+	private Integer getSleepTimeForPolling() {
+		return getIntegerValueOfProperty(Utils.SIS_POLLING_SLEEPTIME);
+	}
+
+	private int getIntegerValueOfProperty(String propertyName) {
```

Comment:
If this is reusable it should be in Utils.  If it is tied to a specific property that should be clear in the name.  One way to make it more generic would be to pass in the property name, the default value, and the error message.